modules/farm_fd2_examples/src/entrypoints/soil_disturbance/App.vue

```diff
@@ -0,0 +1,270 @@
+        <td>required</td>
+        <td>
+          <BFormCheckbox
+            id="required-checkbox"
+            data-cy="required-checkbox"
+            switch
+            v-model="required"
+          />
+        </td>
+      </tr>
+      <tr>
+        <td>showValidityStyling</td>
+        <td>
+          <BFormCheckbox
+            id="styling-checkbox"
+            data-cy="styling-checkbox"
+            switch
+            v-model="validity.showStyling"
+          />
+        </td>
+      </tr>
+      <tr>
+        <td>area</td>
+        <td>
+          <BButton
+            id="set-area-button"
+            data-cy="set-area-button"
+            variant="outline-primary"
+            size="sm"
+            v-on:click="changeArea"
+          >
+            Area = 50
+          </BButton>
+        </td>
+      </tr>
+      <tr>
+        <td>depth</td>
+        <td>
+          <BButton
+            id="set-depth-button"
+            data-cy="set-depth-button"
+            variant="outline-primary"
+            size="sm"
+            v-on:click="changeDepth"
+          >
+            Depth = 6
+          </BButton>
+        </td>
+      </tr>
+      <tr>
+        <td>speed</td>
+        <td>
+          <BButton
+            id="set-speed-button"
+            data-cy="set-speed-button"
+            variant="outline-primary"
+            size="sm"
+            v-on:click="changeSpeed"
+          >
+            Speed = 3
+          </BButton>
+        </td>
+      </tr>
+      <tr>
+        <td>Set Passes</td>
+        <td>
+          <BButton
+            id="set-passes-button"
+            data-cy="set-passes-button"
+            variant="outline-primary"
+            size="sm"
+            v-on:click="changePasses"
+          >
+            Passes = 4
+          </BButton>
+        </td>
+      </tr>
+      <tr>
+        <td>includePasses</td>
+        <td>
+          <BFormCheckbox
+            id="include-passes-checkbox"
+            data-cy="include-passes-checkbox"
+            switch
+            v-model="form.includePasses"
+          />
+        </td>
+      </tr>
+      <tr>
+        <td>Add Planter</td>
+        <td>
+          <BButton
+            id="add-equipment-button"
+            data-cy="add-equipment-button"
+            variant="outline-primary"
+            size="sm"
+            v-on:click="addEquipment"
+          >
+            Add Planter
+          </BButton>
+        </td>
+      </tr>
+      <tr>
+        <td>Clear</td>
+        <td>
+          <BButton
+            id="clear-equipment-button"
+            data-cy="clear-equipment-button"
+            variant="outline-primary"
+            size="sm"
+            v-on:click="clearEquipment"
+          >
+            Clear Equipment
+          </BButton>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+
+  <h5>Component Event Payloads</h5>
+  <table class="example-table">
+    <thead>
+      <th>Event</th>
+      <th>Payload</th>
+    </thead>
+    <tbody>
```

Comment:
Match order of event payloads to the order in which they appear in the form.  And place `valid` at the end to match other examples.